packages/angular/projects/ui-angular/ng-package.json

```diff
@@ -6,9 +6,11 @@
     "entryFile": "src/index.ts"
   },
   "allowedNonPeerDependencies": [
```

Comment:
Can you explain what listing deps here does?

Reply:
"To mitigate the risk of accidentally publishing libraries with dependencies" they make you add dependencies here as well https://github.com/ng-packagr/ng-packagr/blob/main/docs/dependencies.md#allowing-in-the-dependencies-section